From 778255fa51858e3f9a04a7609bab828a2b007f14 Mon Sep 17 00:00:00 2001
From: crazy <crazy@frugalware.org>
Date: Thu, 17 Jan 2019 00:03:53 +0100
Subject: [PATCH 6/6] frugalware drop render group

 We revert these changes so no point to have that group
---
 sysusers.d/basic.conf.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sysusers.d/basic.conf.in b/sysusers.d/basic.conf.in
index 1d26925513..ac3680023d 100644
--- a/sysusers.d/basic.conf.in
+++ b/sysusers.d/basic.conf.in
@@ -28,7 +28,6 @@ g input   120   -            -
 g kvm     -     -            -
 g lp      -     -            -
 g lpadmin 125   -            -
-g render  -     -            -
 g tape    -     -            -
 g video   24    -            -
 g storage 30    -            -
-- 
2.20.1

